### ISSUE TITLE
InputTextArea: Counter disappears after validation

### ISSUE THREAD

**Issue by tarpatel** [primefaces/primefaces#4220]
## 1) Environment
- PrimeFaces version: 6.2.4
- Application server + version: tomcat 8.0.27.0
- Affected browsers: Chrome and Firefox

## 2) Expected behavior

I am using  p:inputTextarea that is set to required=true with h:outputText to show how many characters you have left.

If the user clicks submit and the textarea is empty validation error displays. the counter should still display

![xd2ap](https://user-images.githubusercontent.com/4670516/48148118-9c400c80-e27e-11e8-93be-7a205bc793f4.png)

          
## 3) Actual behavior

when the user clicks submit the character counter disappears.

![lh3ep](https://user-images.githubusercontent.com/4670516/48148121-a19d5700-e27e-11e8-8ad5-e96cbd3f3727.png)

## 4) Steps to reproduce

if you make a page and test the XHTML this should reproducable.

## 5) Sample XHTML

            <h:form id="form">
                <p:messages/>
                <p:inputTextarea id="mytextarea" counter="mycounter" counterTemplate="{0} characters remaining." maxlength="2000" required="true" requiredMessage="mytestarea is required."/>
                <h:outputText id="mycounter"/>
                <br/><br/>
                <p:message for="mytextarea"/>
                <br/><br/>
                <p:commandButton value="save" ajax="false"/>
            </h:form>

## 6) Sample bean

no bean required.

**Comment by tandraschko** [comment]
A PR would be great.

**Comment by tarpatel** [comment]
> A PR would be great.

I am happy to add more info if needed. What is a PR?

**Comment by tandraschko** [comment]
A Pull Request with a bugfix :)

**Comment by melloware** [comment]
@tarpatel I have verified this is already fixed in 6.3 using this attached test case: 
[pf-4220.zip](https://github.com/primefaces/primefaces/files/2713313/pf-4220.zip)

I also verified if I changed the pom.xml from 6.3-SNAPSHOT back to 6.2 the problem existed in 6.2 but is now definitely resolved.

TO run this just run "mvn clean jetty:run-exploded" and navigate to http://localhost:8080/primefaces-test/test.xhtml

You must first build PF 6.3-SNAPSHOT ..

```
git clone https://github.com/primefaces/primefaces.git
cd primefaces
mvn clean install
```
